hudi-cli/src/main/java/org/apache/hudi/cli/commands/SavepointsCommand.java

```diff
@@ -108,9 +113,31 @@ public String rollbackToSavepoint(
       return "Commit " + instantTime + " not found in Commits " + timeline;
     }
 
+    boolean metadataTableEnable = HoodieMetadataConfig.ENABLE.defaultValue();
```

Comment:
this has to go into writeClient.savepoint(). someone could programmatically trigger savepoint as well.

Reply:
Hi @nsivabalan Thanks a lot for your attention. I guess it is to modify `BaseHoodieWriteClient#restoreToSavepoint()`
Changed. PTAL :)